services/github/github-issue-detail.service.js

```diff
@@ -30,7 +140,7 @@ module.exports = class GithubIssueDetail extends LegacyService {
     return {
       base: 'github/issues/detail',
       pattern:
-        ':which(s|state|title|u|author|label|comments|age|last-update)/:user/:repo/:number(0-9+)',
+        ':which(s|state|title|u|author|label|comments|age|last-update)/:user/:repo/:number',
```

Comment:
Two items here:

1) do we want to move to the full words and not promote the one letter aliases (`s` and `u` in particular)? The existing  example below from the legacy service had overridden the pattern, presumably to remove those two shortcuts from the modal window
2) I forgot I had to drop the the `number` param expression, was getting 404 badge not found errors with that in place. Not sure what was going on there 🤔

Reply:
1. Yes to all of that! I was trying to nudge people away from the single-character aliases when I set up the rest of those, and feel like that's a good direction to push in. Using a redirect sounds good.
2. Does it need to be `:number([0-9]+)`?

Reply:
👍